test/crypto-service/test_accountUpdateTransaction.js

```diff
@@ -3,6 +3,17 @@ import mirrorNodeClient from "../../mirrorNodeClient.js";
 import consensusInfoClient from "../../consensusInfoClient.js";
 import { expect, assert } from "chai";
 import { setOperator } from "../../setup_Tests.js";
+import {
+  getEncodedKeyHexFromKeyListConsensus,
+  getPublicKeyFromMirrorNode,
+} from "../../utils/helpers/keys.js";
+
+/* 
+*******************************************************************************************
+You may see "await new Promise((resolve) => setTimeout(resolve, 1000));" in some places of this file.
```

Comment:
This is because of the mirror node right? Can we just do this await logic in the mirror node client itself?

Reply:
Implementing a timeout before each transaction would slow down all the tests. Currently, I've only applied it to a transactions that occasionally fails  due to the **consensus** node not being properly updated.

Reply:
Would it maybe be better, instead of depending on a specific timeout time, to query the state before executing, execute, then repeatedly query again until it updates? And if it doesn't update by the test timeout time then we fail? I'm just thinking if for whatever reason it takes longer than a second to update we could get some flaky results

Reply:
Yes, I am Ok to implement some kind of retry logic.